src/dom/index.js

```diff
@@ -36,7 +36,7 @@ export function setAccessor(node, name, old, value, isSvg) {
 	}
 	else if (name==='style') {
 		if (!value || isString(value) || isString(old)) {
-			node.style.cssText = value || '';
+			node.setAttribute('style', value || '');
```

Comment:
while it would be a smaller line of code, some (admittedly non-rigorous) benchmarking indicates setAttribute is about 30% faster:

http://jsben.ch/GJ0dD

Reply:
setAttribute ignores duplicate assignments, which will never happen here because this function is bypassed when values are equal.

Reply:
Alright I dropped your benchmark into ESBench to double-check, and it shows that both setAttribute and style= perform poorly compared to style.cssText:

https://esbench.com/bench/5bf437e74cd7e6009ef61ce5

<a href="https://esbench.com/bench/5bf437e74cd7e6009ef61ce5"><img width="423" alt="screen shot 2018-11-20 at 11 46 23 am" src="https://user-images.githubusercontent.com/105127/48789017-0662a380-ecba-11e8-9afd-5284c5033478.png"></a>

Reply:
can’t argue there, i’ll go ahead and close